tests/expected/llbc/basic1/expected

```diff
@@ -1,15 +1,15 @@
 fn test::select(@1: bool, @2: i32, @3: i32) -> i32
```

Comment:
In MIR, locals include the return value as well as the function arguments (see the comment on the `locals` field: https://github.com/rust-lang/rust/blob/b0c2d2e5b00339014cba0a31f634af8c146458ed/compiler/stable_mir/src/mir/body.rs#L23). Thus, the function arguments do not include names because they will get the names from the declarations made below. For example, see the MIR for https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b693b59fdc42ef2d5a473a4615f70ba2

Charon has the same behavior, e.g:
```
$ charon --print-llbc
   Compiling sdkfj v0.1.0 (/home/ubuntu/examples/sdkfj)
# Final LLBC before serialization:

fn sdkfj::select(@1: bool, @2: i32, @3: i32) -> i32
{
    let @0: i32; // return
    let s@1: bool; // arg #1
    let x@2: i32; // arg #2
    let y@3: i32; // arg #3
    let @4: bool; // anonymous local

    @4 := copy (s@1)
    if move (@4) {
        @0 := copy (x@2)
    }
    else {
        @0 := copy (y@3)
    }
    drop @4
    return
```